src/app/models/odometer-unit.enum.ts

```diff
@@ -1,4 +1,4 @@
 export enum OdometerReadingUnits {
-  KILOMETERS = 'kilometers',
+  KILOMETERS = 'kilometres',
```

Comment:
Might as well change the enum part too:
```suggestion
  KILOMETRES = 'kilometres',
```